weekly-openmeteo/test_transform.py

```diff
@@ -1 +1,80 @@
 '''Contains tests for transform.py'''
+from datetime import datetime as dt
+import pytest
+from transform import format_time, clean_data, format_data, transform
+
+class TestFormatTime:
```

Comment:
I like the use of test classes, makes it clear what the overarching goal of a group of tests is.

Reply:
I agree